net-irc/weechat/weechat-9999.ebuild

```diff
@@ -11,13 +11,15 @@ inherit cmake lua-single python-single-r1 xdg
 if [[ ${PV} == "9999" ]] ; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/weechat/weechat.git"
+	DOCS="AUTHORS.md CHANGELOG.md CONTRIBUTING.md UPGRADING.md README.md"
 else
 	inherit verify-sig
 	SRC_URI="https://weechat.org/files/src/${P}.tar.xz
 		verify-sig? ( https://weechat.org/files/src/${P}.tar.xz.asc )"
 	VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/weechat.org.asc
 	BDEPEND+="verify-sig? ( sec-keys/openpgp-keys-weechat )"
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86 ~x64-macos"
+	DOCS="AUTHORS.adoc ChangeLog.adoc Contributing.adoc ReleaseNotes.adoc README.adoc"
```

Comment:
Given future versions won't have these filenames, I'd just fix the version below instead (don't put it in the if/else).

Reply:
Sure, I can do that, just wanted to be safe in case that there would be new releases for 4.3 or 4.2 branches.